### DISCUSSION TITLE
oras push command with --manifest-config & --manifest-annotations options is not working

### DISCUSSION THREAD

**Discussion by bthunuguntla** [oras-project/oras#576] [Q&A] [answered]
I installed ORAS 0.14.1 referring to the URL [https://oras.land/cli/](url) successfully.

I also pushed & pulled artifact to & from Azure Container Registry using the below commands successfully.

`oras push myacr.azurecr.io/devops/projectname/cli/linux:474438 D:\home\vsts\work\1\a\projectname\publish\linux-x64\projectcfg`

`oras pull myacr.azurecr.io/devops/projectname/cli/linux:474388`

I tried the following commands to pass manifest config or annotations along with oras push, referring to the below URLs
[https://oras.land/cli/3_manifest_config/](url)
[https://oras.land/cli/4_manifest_annotations/](url)

`oras push --manifest-config D:\runtimeLinux64-annotations.json myacr.azurecr.io/devops/projectname/cli/linux:474438 D:\home\vsts\work\1\a\projectname\publish\linux-x64\projectcfg`

`oras push --manifest-annotations D:\runtimeLinux64-annotations.json myacr.azurecr.io/devops/projectname/cli/linux:474438 D:\home\vsts\work\1\a\projectname\publish\linux-x64\projectnamecfg`

But I got the following errors.

**Error: unknown flag: --manifest-config**
**Error: unknown flag: --manifest-annotations**

What am I missing here?
Any help is very much appreciated.

**Comment by shizhMSFT** [comment, marked as answer]
@bthunuguntla There is a breaking change in [v0.14.0](https://github.com/oras-project/oras/releases/tag/v0.14.0). Precisely,
* `--manifest-config` is renamed to `--config`
* `--manifest-annotations` is renamed to `--annotation-file`

@FeynmanZhou is working on updating those outdated documentation on the `oras.land` website.
